src/Configuration/GrumPHP.php

```diff
@@ -46,6 +46,14 @@ public function getHooksPreset(): string
         return $this->container->getParameter('hooks_preset');
     }
 
+    /**
+     * @return array
+     */
+    public function getGitHookVariables()
```

Comment:
Remove docblock and add return type array

Reply:
Okay